playground/utils/exportFiles.tsx

```diff
@@ -68,17 +56,103 @@ const indexHTML = (tabs: Tab[]) => dedent`
 </html>
 `;
 
+/**
+ * This function extracts the name and version of an imported
+ * package, returning an array [name, version], in case the
+ * import si written as name@version, or otherwhise as [name, '']
+ */
+ function extractNameAndVersionFromImportPath(importPath: string): string[] {
+    const withoutCDN = importPath.startsWith(CDN_URL.domain)
+      ? importPath.split(CDN_URL.domain)[1]?.split(CDN_URL.mode)[0]
+      : importPath
+    const withoutCDNSplitted = withoutCDN.split('/')
+
+    const rawName = withoutCDN.startsWith('@')
+      ? `${withoutCDNSplitted[0]}/${withoutCDNSplitted[1]}`
+      : withoutCDNSplitted[0]
+    const rawNameSplitted = rawName.split('@')
+
+    const name = rawName.startsWith('@')
+      ? `@${rawNameSplitted[1]}`
+      : rawNameSplitted[0]
+
+    const version = name.startsWith('@')
+      ? (rawNameSplitted[2] ?? '')
+      : (rawNameSplitted[1] ?? '')
+
+    return [name, version]
+  }
+
+/**
+ * This function sends a request to esm.sh in order to get
+ * the latest version of a pkg, which will be included in
+ * the URL to which esm.sh automatically redirects
+ */
+  function getLatestVersion(pkgName: string): Promise<string[]> {
+    return fetch(MAKE_CDN_URL(pkgName))
+      .then(res =>
+        extractNameAndVersionFromImportPath(res.url)[1].length ?
+          extractNameAndVersionFromImportPath(res.url)
+        :
+          [pkgName, 'latest']
+      )
+      .catch(() => [pkgName, 'latest'])
+  }
```

Comment:
adapted to the new CDN.
This is actually faster than the function used with Skypack